src/NodaTime.Test/PeriodTest.cs

```diff
@@ -931,6 +940,39 @@ public void Comparer_NonDurationablePeriods()
             Assert.AreEqual(0, februaryComparer.Compare(month, month));
         }
 
+        public static TestCaseData[] PeriodMaxAndMinValues
```

Comment:
This can be simplified with an expression-bodied member, and it can be private:

```csharp
private static TestCaseData[] PeriodMaxAndMinValues =>
[
    new TestCaseData(Period.MaxValue, int.MaxValue, long.MaxValue).SetName("All members should be MaxValue"), 
    new TestCaseData(Period.MinValue, int.MinValue, long.MinValue).SetName("All members should be MinValue")
];
```